covid19-etl/etl/dsci.py

```diff
@@ -0,0 +1,91 @@
+import csv
+
+from etl import base
+from helper.metadata_helper import MetadataHelper
+
+
+class DS4C(base.BaseETL):
+    def __init__(self, base_url, access_token):
+        super().__init__(base_url, access_token)
```

Comment:
will need to pull changes from master and add the `s3_bucket` param 😬

Reply:
Easy. Done. 👍